mk_runtool.footer

```diff
@@ -10,4 +10,57 @@ for toolname in list_tools():
     setattr(sys.modules[__name__], toolname, make_tool(toolname))
     __all__.append(toolname)
 
+# Now, some tools added that don't use par files and are thus not
```

Comment:
We could add a marker to these scripts [*], so this would change the onus from this script to whoever writes the script-with-no-parameter file. But I'm not sure it's worth it (i.e. if we do this then it's probably best to have the logic here).

[*] ie some easily greppable token

Reply:
Yeah, but then we have to make sure that marker is in each of them, which seems like a thing we'll forget about two years form now and we can't automatically enforce without even more complex logic. We also *could* have the logic from [the codeblock I used to identify those scripts](https://github.com/cxcsds/ciao-contrib/issues/643#issue-1419049281) here, but I decided that the risk of including too much of that accidentally is too high, e.g. I don't think `install_marx` makes much sense here.

I could add a check to make sure that anything added in this stage is not in `sys.modules[__name__]` already, in case we ever add a par file to one of the scripts that don't have it right now.

Reply:
My impression is that this is list is pretty stable because (a) we don't add scripts all that often and (b) when we do most of them use parameter files, so I suggest to start with the simple thing (a hardcoded list) for now. If we find ourselves changing that list often, then we can go and do something more complicated later.